src/main/java/Lesson4/Task1/Main.java

```diff
@@ -0,0 +1,26 @@
+package Lesson4.Task1;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        AbstractHandler handler;
+        System.out.println("Enter document type:");
+        String type = scan.next();
+        switch (type){
+            case "XML": handler = new XMLHandler();
+                break;
+            case "TXT": handler = new TXTHandler();
+                break;
+            case "DOC": handler = new DOCHAndler();
+                break;
+            default: handler = new XMLHandler();
+                System.out.println("Default file");
```

Comment:
Why did you decide that XML is default one?
Here should not be format by default (as it was not described in task), just error message like "Document of this format can't be proceed"